tests/qa_automation/patch_and_reboot.pm

```diff
@@ -41,9 +42,17 @@ sub run {
 
     add_test_repositories;
 
+    # JeOS is a bootable image and doesn't have installation where we can install
+    #   install updates as for SLE DVD installation, so we need to update manually.
+    if (is_jeos) {
+        record_info('Updates', script_output('zypper lu'));
+        zypper_call('up', timeout => 300);
+    }
+
     fully_patch_system;
```

Comment:
Do we still need `fully_patch_system` if we are running a `zypper up` before? Perhaps putting this into a `else` block makes sense here.

Reply:
I left it intact "just in case" but after looking at the tests, this step really does nothing, so better to skip it. 
http://fromm.arch.suse.de/tests/1314#step/patch_and_reboot/167